django/db/models/functions/window.py

```diff
@@ -0,0 +1,120 @@
+from django.db.models import Aggregate, FloatField, IntegerField
+
+__all__ = [
+    'RowNumber', 'Rank', 'DenseRank', 'Ntile', 'FirstValue',
+    'PercentRank', 'NthValue', 'LastValue', 'CumeDist', 'Lag',
+    'Lead',
+]
+
+
+class CumeDist(Aggregate):
```

Comment:
A few of them can be used in `WITHIN GROUP`, and thus fall under the category "ordered set aggregates".

https://www.postgresql.org/docs/current/static/functions-aggregate.html#FUNCTIONS-HYPOTHETICAL-TABLE